docs/calicoctl/status.md

```diff
@@ -0,0 +1,59 @@
+
+# User guide for 'calicoctl status' commands
+
+This sections describes the `calicoctl status` commands.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl status' commands
+
+Run
+
+    calicoctl status --help
+
+to display the following help menu for the calicoctl status commands.
+
+```
+
+Usage:
+  calicoctl status
+
+Description:
+  Print current status information regarding calico-node container
+  and the BIRD routing daemon.
+
+```
+
+## calicoctl status commands
+
+
+### calicoctl status
+This command shows the running state and uptime of the calico-node container.  
+It also shows the BGP peer state information from the BIRD routing daemon.
```

Comment:
nicer to list what the status shows rather than splitting into sentences like this.